### PR TITLE
clear up and init qwoutputlayer qwtearingcontrolv1

### PR DESCRIPTION
**Pull request by wineee**
closes: https://github.com/vioken/qwlroots/issues/201 https://github.com/vioken/qwlroots/issues/213

### CODE REVIEW

File: src/types/qwswitch.h [hunk 1]
```diff
@@ -1 +1,3 @@
 #include <qwinputdevice.h>
```
Comment (wineee) [inline src/types/qwswitch.h:1]: 忘了删除

File: src/types/qwoutputlayer.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2024 rewine <luhongxu@deepin.org>.
```
Comment (zccrs) [inline src/types/qwoutputlayer.h:1]: UnionTech Software Technology Co., Ltd.

**Review by zccrs — CHANGES_REQUESTED** [review]

File: src/types/qwoutputlayer.h [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#include <wlr/types/wlr_output_layer.h>
+}
+
+QW_BEGIN_NAMESPACE
+
+class QW_CLASS_OBJECT(output_layer)
+{
+    QW_OBJECT
+    Q_OBJECT
+    QW_SIGNAL(feedback, wlr_output_layer_feedback_event*)
```
Comment (zccrs) [inline src/types/qwoutputlayer.h:18]: add a new line between of Q_OBJECT and QW_SIGNAL

File: src/types/qwgammacontorlv1.h [hunk 4]
```diff
@@ -20,6 +20,14 @@ class QW_CLASS_OBJECT(gamma_control_manager_v1)
 
 public:
     QW_FUNC_STATIC(gamma_control_manager_v1, create)
+    QW_FUNC_MEMBER(gamma_control_manager_v1, get_control)
```
Comment (zccrs) [inline src/types/qwgammacontorlv1.h:23]: add a new line between of QW_FUNC_STATIC and QW_FUNC_MEMBER code blocks.

**Review by zccrs — CHANGES_REQUESTED** [review]

**Pull request merged**